index.js

```diff
@@ -395,20 +399,148 @@ vorpal
         });
       },
       function(passphrase, seriesCb){
-        var delegate = args.name;
-        var transaction = arkjs.delegates.createVote(passphrase);
+        var delegateName = args.name;
+        arkjs.crypto.setNetworkVersion(30);
```

Comment:
no here you should use `arkjs.crypto.setNetworkVersion(network.config.version);` instead